README.md

```diff
@@ -72,6 +72,7 @@ Installation instructions for previous releases can be found in [Metrics Server
 
 Metrics Server | Metrics API group/version | Supported Kubernetes version
 ---------------|---------------------------|-----------------------------
+0.8.x          | `metrics.k8s.io/v1beta1`  | 1.19+
```

Comment:
I think we should update the supported version to latest ones we test as we are not testing in this kubernetes version anymore. As end to end test runs 1.31 and above 1.31 is good to add here?